src/jsx.ts

```diff
@@ -18,13 +20,16 @@ export interface ParseContext {
   mode: ParseMode
 }
 
-export function JSXSlack(
-  node: JSXSlack.Node,
-  context: ParseContext = {
```

Comment:
An exported `JSXSlack` no longer have second argument for private use. We updated to use this in internal `parseJSX` function.